sources/packages/web/src/services/EducationProgramService.ts

```diff
@@ -23,17 +23,17 @@ export class EducationProgramService {
     return this.instance || (this.instance = new this());
   }
 
-  public async getProgram(programId: number): Promise<ProgramDto> {
+  public async getProgram(programId: number): Promise<EducationProgramAPIDTO> {
```

Comment:
The enum name is missing the In/Out.

Reply:
The one I renamed was used for In/Out.

Reply:
As we talked about, we can handle it during the API reorganization.